.github/workflows/release.yml

```diff
@@ -58,6 +58,12 @@ jobs:
           name: SDK_Artifacts_C
           path: libxenserver/usr/local/
 
+      - name: Retrieve Java SDK distribution artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: SDK_Artifacts_Java
+          path: dist/
+
```

Comment:
Will you add the package and release steps in following code in future?

Reply:
files are indeed packaged with the release. see further down on line `105`, where we store everything in `dist/*`
```yml
[...]
gh release create ${{ github.ref_name }} --repo ${{ github.repository }} --generate-notes dist/* \
```

Reply:
also if you're curious here's an example release:

https://github.com/danilo-delbusso/xen-api/releases/tag/v60.1.3

Reply:
You may want to add the src and doc jars too so we have the full set. Also make the `PRERELEASE` bit lowercase to match the other packages with `prerelease` in the name.